delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/ResourceF.scala

```diff
@@ -84,9 +88,44 @@ object ResourceF {
       r.void.asJsonObject deepMerge r.value.asJsonObject
     }
 
+  private def resourceFUnitJsonLdEncoder(
+      valueContext: ContextValue
+  )(implicit base: BaseUri): JsonLdEncoder[ResourceF[Unit]] =
+    JsonLdEncoder.computeFromCirce(
+      _.id.resolvedAgainst(base.endpoint.toIri),
+      ContextValue(contexts.metadata).merge(valueContext)
```

Comment:
if this method is called just within 

```scala
val rfUnitJsonLdEncoder = resourceFUnitJsonLdEncoder(context(value))
```

We don't need to add again  the contexts.metadata, since `context` method already has it. We can just use `valueContext` directly